packages/discovery-provider/src/api/v1/users.py

```diff
@@ -884,6 +889,166 @@ def get(self, id):
         return success_response(favorites)
 
 
+USER_PLAYLISTS_ROUTE = "/<string:id>/playlists"
+
+
+playlists_response_full = make_full_response(
+    "playlists_response_full", full_ns, fields.List(fields.Nested(full_playlist_model))
+)
+
+
+@full_ns.route(USER_PLAYLISTS_ROUTE, doc=False)
+class PlaylistsFull(Resource):
+    def _get(self, id, authed_user_id):
+        decoded_id = decode_with_abort(id, ns)
+        args = user_playlists_route_parser.parse_args()
+
+        current_user_id = get_current_user_id(args)
+
+        offset = format_offset(args)
+        limit = format_limit(args)
+
+        args = GetPlaylistsArgs(
+            user_id=decoded_id,
+            authed_user_id=authed_user_id,
+            current_user_id=current_user_id,
+            filter_deleted=True,
+            limit=limit,
+            offset=offset,
+            kind="Playlist",
+        )
+        playlists = get_playlists(args)
+
+        def format_playlist(playlist):
```

Comment:
ick 😢 

Could this happen inside of `extend_playlist` or no?

Reply:
I just copied what we did elsewhere... I'd be a little worried about changing the whole thing and breaking something else, but i can give it a go and test around

Reply:
nah i think be consistent is best

Reply:
actually it's not so bad. i think it should be ok - i spot checked the other usages, and I think the marshaling will take care of what we need elsewhere where the activity_timestamps are used